ios/FluentUI/People Picker/PersonaCell.swift

```diff
@@ -29,6 +29,23 @@ open class PersonaCell: TableViewCell {
             avatar.state.image = image
         }
 
+        if let loadImageAsync = persona.loadImageAsync {
+            loadImageAsync { [weak avatar] image in
+                guard let avatar = avatar,
+                      let image = image else {
+                    return
+                }
+
+                if Thread.isMainThread {
+                    avatar.state.image = image
+                } else {
+                    DispatchQueue.main.async {
+                        avatar.state.image = image
```

Comment:
when we send over image and avatar to the next main queue, I still think we should weak capture here tool

Reply:
I could include the weak capture in `DispatchQueue.main.async`, but logically, I don't see the need for it. The intend of weak capture in `loadImageAsync` closure is to prevent avatar living longer than needed, retain cycle shouldn't be an issue here. if `loadImageAsync` completed and `avatar` is still in memory, then `DispatchQueue.main.async` is not going to keep it for any longer since the DispatchQueue closure will get executed soon. Correct me if I missed something.